src/components/TimeseriesChart.js

```diff
@@ -712,6 +768,81 @@ function mapStateToProps(state) {
   };
 }
 
+function createVerticalLine(
```

Comment:
Misschien kunnen we al deze functies beter inlinen in de Array.forEach of Array.map. Maar misschien ook niet. Hier kunnen we even over speculeren.